.github/pull_request_template.md

```diff
@@ -1,18 +1,26 @@
+Please click the `Preview` tab and select a different PR template if your change isn't for an                                            existing web component.
+
+[PIE Docs Template](?expand=1&template=docs_template.md)
+[New Web Component Template](?expand=1&template=new_component_template.md)
+
+═══════════════════════════════════════════════════════════
+
 ## Describe your changes (can list changeset entries if preferable)
 
 
 ## Author Checklist (complete before requesting a review)
 - [ ] I have performed a self-review of my code
-- [ ] If it is a core feature, I have added thorough tests
-- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
-- [ ] If it is a component change, I have reviewed the Storybook preview
-- [ ] If there are visual test updates, I have reviewed them properly before approving
+- [ ] I have added thorough tests where applicable (unit / component / visual).
+- [ ] I have reviewed the Storybook preview
+- [ ] I have reviewed visual test updates properly before approving
+- [ ] If changes will affect consumers of the package, I have created a changeset entry.
+- [ ] If a changeset file has been created, I have used the `/snapit` functionality to test my changes in a consuming application.
 
 ## Reviewer checklists (complete before approving)
 ### Reviewer 1
-- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
+- [ ] I have reviewed the Storybook preview
```

Comment:
if this is a general purpose template, is there a reason to mention storybook but not docs site?

Reply:
The general purpose is for existing Web Components. PIE Docs has it's own template :)

Reply:
Perhaps we need to be more granular in that case? I imagine it would get deleted a lot for non-component changes like configs, chores, icons etc

Reply:
Maybe could just update this to:

```suggestion
- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
```

And then PR authors can amend/delete as needed (as existing components should update PIE docs if there's a change in the way the component works, as README stuff has been ported to the docs site now)

Reply:
All done!